test/capybara_integration_test_case.rb

```diff
@@ -56,6 +78,20 @@ class CapybaraIntegrationTestCase < ActionDispatch::IntegrationTest
   include GeneralExtensions
   include FlashExtensions
   include CapybaraSessionExtensions
+  include CapybaraMacros
+
+  # Javascript tests use this
+  Capybara.register_driver(:firefox_headless) do |app|
```

Comment:
I'm struggling to find if and where this is actually used.  Are there any good examples showing how it works?

Reply:
Not used yet. Would like to keep it in so i can start writing with it... 
Note that this just registers the driver. As long as the driver isn't loaded in a test it won't slow down the tests.